docs/tutorial/replica-setup-rds.md

```diff
@@ -7,10 +7,10 @@ MyDuck Server supports setting up replication from common cloud-based MySQL offe
 Ensure the necessary parameters are properly set. Run the following query to check the configuration:
```

Comment:
`and self-hosted MySQL Community Edition` -> `self-hosted MySQL Community Edition, and self-hosted MariaDB`.